src/ex1/styles/ids.css

```diff
@@ -0,0 +1,59 @@
+#add-todo {
+  display: flex;
+}
+
+#new-todo-title:focus {
+  border: 2px solid var(--very-dark-green);
+  border-radius: 4px;
+  outline: none;
+}
+
+#new-todo-title {
+  padding: 5px 10px;
+  font-family: 'Indie Flower', cursive;
```

Comment:
It's a good practice to define the main font in the HTML to avoid re-assigning it on each text field

Reply:
I use two fonts. Now I defined one of them in HTML and assign the second one if necessary. But for some reason the font from HTML is not applied to the input textbox and buttons. So I decided that the "main" font is the one which is not supposed to be in forms and buttons because there I need to re-assign the font anyway.